### PR TITLE
utils: update `accuracy` for classification task

### PR DESCRIPTION
**Pull request by iblis17**

### CODE REVIEW

**Comment by iblis17** [comment]
BTW, can we import model-zoo as a submodule? then we can freeze the correspond version of model-zoo

File: src/utils.jl [hunk 1]
```diff
@@ -28,15 +28,31 @@ end
...
-  n = 0
-  correct = 0
-  for (x, y) in data
-    x, y = tobatch.((x, y))
-    n += size(x, 1)
-    correct += sum(argmax(m(x)) .== argmax(y))
-  end
-  return correct/n
+"""
+    accuracy(m, pred, label)
```
Comment (CarloLucibello) [inline src/utils.jl]: outclass missing

**Comment by MikeInnes** [comment]
So I think the original idea was this was for it to work with a dataset exactly like you'd give `train!`, to make things convenient in that case.

That said I'm wondering if Flux should really include this at all... it might just be confusing to people if it doesn't apply to their specific dataset / setup.

**Comment by iblis17** [comment]
I'm waiting for [MLMetrics](https://github.com/JuliaML/MLMetrics.jl) getting released.
And I'm comfortable with my MNIST model now: https://github.com/FluxML/model-zoo/pull/7#issuecomment-339611574

**Pull request closed without merging**